README.md

```diff
@@ -25,11 +25,10 @@ use Shopgate\ConnectSdk\Services\Events\Client;
 use Shopgate\ConnectSdk\Services\Events\DTO\V1\Payload\Catalog\Category as CategoryDto;
 
 $config = [
-    'http' => [
-        'headers'      => ['Authorization' => 'Bearer XXX'],
-        'merchantCode' => 'EE1',
-        'service'      => 'omni-event-receiver'
-    ]
+    'merchantCode'  => 'EE1',
+    'service'       => 'omni-event-receiver',
```

Comment:
The service will be handled internally and depends on the call. Basically the dev should not know about it, so can you please remove this from example? Same for line 60